nucliadb_node/src/grpc/grpc_writer.rs

```diff
@@ -381,6 +381,16 @@ impl NodeWriter for NodeWriterGRPCDriver {
 
     async fn merge(&self, request: Request<ShardId>) -> Result<Response<MergeResponse>, Status> {
         let shard_id = request.into_inner().id;
+        let shard_id_copy = shard_id.clone();
+        let shards_tasks_copy = Arc::clone(&self.shards);
+        let load_shard_task = || obtain_shard(shards_tasks_copy, shard_id_copy);
```

Comment:
Could you add a comment explaining the load shard from cache hack?

Reply:
true!